### ISSUE TITLE
Multiple packages matched with different SystemSKU

### ISSUE THREAD

**Issue by tonycontorno** [MSEndpointMgr/ConfigMgr#277]
Hello,

My apologies if this has already been brought up, but I am having an issue regarding Microsoft Surface 4 devices.  I am receiving multiple matches despite the fact that the descriptions clearly don't have the Surface_Pro_4 listing in the description field.  I am at a bit of a loss as to why that is happening.  If anyone else has seen this behavior before and how to fix it, I would greatly appreciate some guidance.

Currently on 4.0.3 of the MDM solution, and 2002 for MECM.

![image](https://user-images.githubusercontent.com/72424709/95151137-7fa0fd80-074f-11eb-8621-c36f70efe5e1.png)

![image](https://user-images.githubusercontent.com/72424709/95151115-74e66880-074f-11eb-846a-0b1abbde8ecb.png)

**Comment by tonycontorno** [comment]
To add to it, I tried manually recreate one of the packages to see if maybe it was something odd with one of the automated packages, but it was returned as a possible driver package again.

**Comment by simulacra75** [comment]
Seeing the same behaviour here when using script versions 3.08 and 3.09 with DAT ver. 6.4. 9

**Comment by shaikh441** [comment]
Currently using latest MDM and PS script (SCCM 2004) and seeing exact same thing.

**Comment by shaikh441** [comment]
Any potential fixes to this as we are having to revert to the legacy driver packages for Surface devices as the latest tool/script will not work with Surface Pro 6 and 7 (not tested on earlier models yet)

**Comment by Ar5had07** [comment]
Yeah it's been a known issue for a very long time, I am using an old script that still works but doesn't work with WIM & 7-Zip Compression and doesn't work over CMG.  Works internally fine though.  The script that works is version 3.04.  So I have 2 scripts in my TS, 1 for all Surfaces and the other for the rest.  I have tweeted at them, but no replies.

**Comment by shaikh441** [comment]
Thanks for letting me know. 
I am currently putting in a wmi query to bypass the Surfaces when applying MDM drivers and use legacy driver packages for all surface devices only for now.

**Comment by azulol** [comment]
We have the same problem with script version 3.0.4. on the Lenovo T14s 20UJ (AMD Modell).

Has someone solved the issue?

Right now we are working with the wmi query and a driver package for this Modell to solve it for now.
